packages/widget_toolkit/lib/src/lib_edit_address/views/edit_address_page.dart

```diff
@@ -122,87 +95,83 @@ class EditAddressPage<T extends PickerItemModel> extends StatelessWidget {
-                        return context
-                            .read<EditAddressBlocType>()
-                            .events
-                            .setAddress(addressModel);
-                      },
-                          translateError: translateError,
-                          validator: validator,
-                          countryCustomIcon: countryCustomIcon,
-                          editCountryFieldType: editCountryFieldType,
-                          cityCustomIcon: cityCustomIcon,
-                          editCityFieldType: editCityFieldType,
-                          addressCustomIcon: addressCustomIcon,
-                          editAddressFieldType: editAddressFieldType,
-                          searchCountryService: searchCountryService,
-                          editAddressLocalizedStrings:
-                              editAddressLocalizedStrings,
-                          editAddressService: editAddressService,
-                          searchCountryCustomBuilders:
-                              searchCountryCustomBuilders,
-                          textFieldsModalConfiguration:
-                              textFieldsModalConfiguration,
-                          countryPickerModalConfiguration:
-                              countryPickerModalConfiguration),
-                    if (!snapshot.hasData)
-                      SizedBox(height: context.editAddressTheme.spacingXL),
-                    if (!snapshot.hasData)
-                      RxBlocBuilder<EditAddressBlocType, bool>(
-                        state: (bloc) => bloc.states.isLoading,
-                        builder: (context, isLoadingSnapshot, bloc) =>
-                            RxBlocBuilder<EditAddressBlocType, AddressModel>(
-                          state: (bloc) => bloc.states.onAddressSet,
-                          builder: (context, address, bloc) =>
-                              GradientFillButton(
-                            elevation: 0,
-                            text: buttonText,
-                            state: isLoadingSnapshot.data ?? false
-                                ? ButtonStateModel.loading
-                                : ButtonStateModel.enabled,
-                            areIconsClose: true,
-                            onPressed: () async {
-                              if (saveAddress == null) {
-                                context
-                                    .read<EditAddressBlocType>()
-                                    .events
-                                    .saveAddress();
-                              } else {
-                                if (address.hasData) {
-                                  saveAddress?.call(address.data!);
+                      if (!snapshot.hasData)
+                        EditAddressForm.withDependencies<T>(
+                            context, addressModel, (addressModel) {
+                          return context
```

Comment:
For better readability you can use an expression body

Reply:
The `return` was changed to expression body here and on some other places